File: UnixLab3/src/Python/main.py
from sys import argv

def mltp(first, second):
    return first * second

def main():
    if len(argv) < 3:
        print('Not enough args!')
    else:
        first = float(argv[1])
        second = float(argv[2])

        print(f"{first} * {second} = {mltp(first, second)}")

if __name__ == '__main__':
    main()

File: UnixLab3/src/JS/main.js
const { argv } = require('./argv');

const mltp = (first, second) => {
    return first * second;
};

if (argv.length < 2) {
    console.log('Not enough args!');
} else {
    const first = Number(argv[0]);
    const second = Number(argv[1]);
    
    console.log(`${first} * ${second} = ${mltp(first, second)}`);
}